.vscode/cirkopt.code-workspace

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
We should maybe add a `.vscode/*` to the `.gitignore` and not commit these files